nmpolicy/internal/resolver/resolver_test.go

```diff
@@ -708,3 +709,47 @@ base-iface-routes:
 		runTest(t, testToRun)
 	})
 }
+
+func testReplaceWithoutCaptureRef(t *testing.T) {
+	t.Run("Filter list with capture reference", func(t *testing.T) {
+		testToRun := test{
+			captureASTPool: `
+bridge-routes:
+  pos: 1
+  replace:
+  - pos: 2
+    identity: currentState
+  - pos: 3
+    path:
+    - pos: 4
+      identity: routes
+    - pos: 5
+      identity: running
+    - pos: 6
+      identity: next-hop-interface
+  - pos: 7
+    string: br1
+`,
+
+			expectedCapturedStates: `
+bridge-routes:
+  state:
+    routes:
+      running:
+      - destination: 0.0.0.0/0
+        next-hop-address: 192.168.100.1
+        next-hop-interface: br1
```

Comment:
Please add an entry with `next-hop-interface` different than `br1`. I believe the current code has a bug a this entry will be filtered.

Reply:
The input currentState has no route with br1, where do I have to add it ?